centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import { useEffect, useState } from 'react';
+
+import { Meta, StoryObj } from '@storybook/react';
+
+// import { expect } from '@storybook/jest';
```

Comment:
Will we need that in the future?

Reply:
this specific one not, been removed

in the `front_src` there are chunks of code that are temporarily commented out, they should be marked with a FIXME / TODO if it's intentional / for future unfinished work